docs-src/docs/kognic-io/scenes/lidars_with_imu_data.md

```diff
@@ -70,18 +70,28 @@ client = KognicIOClient()
 client.lidars_and_cameras_sequence.create(
     lidars_and_cam_seq,
     project="project-ext-id",
-    dryrun=False
+    dryrun=True,
 )
 ```
 
-:::tip Use dryrun to validate 
+:::tip Use dryrun to validate
 Setting `dryrun` parameter to true in the method call, will validate the scene using the API but not create it.
 :::
 
 ## Enable/disable motion compensation
 
-By default motion compensation is performed for s with LIDAR pointclouds when IMU data is provided.
+By default motion compensation is performed for scenes with LIDAR pointclouds when IMU data is provided.
+
+Whether motion compensation is enabled is controlled by an [feature flag](../feature_flags), the default is enabled. To disable motion compensation, you must explicitly provide an empty feature flag
```

Comment:
```suggestion
Whether motion compensation is enabled or not is controlled by a [feature flag](../feature_flags). By default it is enabled but it can be disabled by providing an empty feature flag.
```